server/package.json

```diff
@@ -15,7 +15,10 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "typeorm": "npx typeorm-ts-node-commonjs",
+    "migration:create": "./create-migration.sh",
```

Comment:
It seems that if we want to use this like "npm run migration:create <filename>" we need to use this kind of additional script. One way I found to do it without a file is with a function:

`"migration:create": "func() { npm run typeorm migration:create src/database/migrations/$1 ;}; func",`

Reply:
This is much cleaner. I'll update the PR mometarily.